nrm_django/grants/admin.py

```diff
@@ -39,131 +39,157 @@ class GrantAdmin(admin.ModelAdmin):
         "created_in_instance",
         "modified_date",
         "status",
+        "wppp_status",
     ]
-    fieldsets = (
+    fieldsets = [
         (
-            "Required",
+            "",
             {
                 "fields": (
-                    ("applicant_name", "proj_cfda_no"),
-                    ("application_type", "app_submission_type"),
-                    ("app_submit_date", "app_received_date"),
+                    "proj_title",
                     ("proposed_start_date", "proposed_end_date"),
-                    ("proj_title", "status"),
-                    "progrm_responsibility_type",
-                )
-            },
-        ),
-        (
-            "Details",
-            {
-                "fields": (
-                    ("gid",),
-                    ("proj_desc", "areas_effected"),
+                    ("applicant_name",),
+                    # Org goes here
```

Comment:
We think org will come from contacts. Maybe.

Reply:
This is getting hooked up in https://github.com/18F/NRM-Grants-Agreements/tree/156-contacts